lib/Model.php

```diff
@@ -234,7 +235,7 @@ class Model
 	 * Constructs a model.
 	 *
 	 * When a user instantiates a new object (e.g.: it was not ActiveRecord that instantiated via a find)
-	 * then @var $attributes will be mapped according to the schema's defaults. Otherwise, the given
+	 * then <code>$attributes</code> be mapped according to the schema's defaults. Otherwise, the given
```

Comment:
Ah yeah. The reason i fixed this one was `@var $attributes will` was considered `{@var $attributes will}` and actually affected `string $attribute	Name of the attribute` in a very odd way. Goal here is to not rewrite the docblocks, just fix them from causing IDE warnings when using the methods.